### PR TITLE
Add subtitles and GitHub links

### PR DESCRIPTION
**Pull request by sarayourfriend**

### CODE REVIEW

File: src/github.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <a href="https://github.com/SeattleDSA/parler-data-searcher/">
+            <p>Parler Data Searcher</p>
+            <p class="subtitle">Search frontend for our MongoDB stored scrape of Parler</p>
+        </a>
+
+        <h2>Bots 🤖</h2>
+        <a href="https://github.com/SeattleDSA/signal-scanner-bot">
+            <p>Signal Scanner Bot</p>
+            <p class="subtitle">This bot monitors the DSA's Scanner signal channel and forwards relevant police comm messages to our twitter account</p>
+        </a>
```
Comment (AetherUnbound) [inline src/github.html]: Sorry!! This one is DSA specific and we don't advertise it publicly. We could add this one though: https://github.com/SeattleDSA/umbrella-openmhz-encrypted. Blurb:

> A bot that tweets aggregated graphs for encrypted calls on OpenMHZ

File: src/github.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <a href="https://github.com/SeattleDSA/encryptedbot_py">
+            <p>Encrypted Bot dot py</p>
+            <p class="subtitle">A bot that tweets every time calls are made on radio monitored by OpenMHZ</p>
```
Comment (AetherUnbound) [inline src/github.html]:
```suggestion
            <p class="subtitle">A bot that tweets every time encrypted calls are made on radio monitored by OpenMHZ</p>
```

File: src/index.html [hunk 3]
```diff
@@ -15,27 +15,66 @@
...
+            <p class="subtitle">Shit post with us</p>
+        </a>
+        <a href="mailto:techblocsea@protonmail.com">
+            <p>Proton Mail</p>
+            <p class="subtitle">Send us an old-fashioned email</p>
+        </a>
+        <a href="/github.html">
+            <p>Open Source Projects</p>
+            <p class="subtitle">Help us build the technology for a future without cops</p>
+        </a>
```
Comment (AetherUnbound) [inline src/index.html:29]: LOVE

File: src/index.html [hunk 4]
```diff
@@ -15,27 +15,66 @@
...
-        <a href="https://pra-tracker.tech-bloc-sea.dev/">Public Records Act Request Tracker</a>
-        <a href="https://radio-chaser.tech-bloc-sea.dev/">Radio Chaser</a>
-        <a href="https://parler-searcher.tech-bloc-sea.dev/">Parler Data Searcher</a>
+        <a href="http://1-312-hows-my-driving.glitch.me/">
+            <p>1-312-hows-my-driving</p>
+            <p class="subtitle">Look up cops and their cars</p>
+        </a>
+        <a href="https://openoversight.tech-bloc-sea.dev/">
+            <p>OpenOversight</p>
+            <p class="subtitle">Help your community identify cops and hold them accountable</p>
```
Comment (AetherUnbound) [inline src/index.html]:
```suggestion
            <p class="subtitle">Find in-depth data and photos for your local <strike>tormentors</strike> officers</p>
```

File: src/index.html [hunk 5]
```diff
@@ -15,27 +15,66 @@
...
+        <a href="https://parler-searcher.tech-bloc-sea.dev/">
+            <p>Parler Data Searcher</p>
+            <p class="subtitle">Search the infamous Parler data scrape</p>
+        </a>
 
         <h2>Bots 🤖</h2>
-        <a href="https://twitter.com/watching_spd">Encrypted Chatter</a>
+        <a href="https://twitter.com/watching_spd">
+            <p>Encrypted Chatter</p>
+            <p class="subtitle">Find out when cops don't want you to know what they're saying</p>
```
Comment (AetherUnbound) [inline src/index.html:70]: :heart: your descriptions

**Review by AetherUnbound — COMMENTED** [review]
GREAT stuff, a few wording comments! I like how the github link is in the get involved section, that's solid.

File: src/github.html [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+            <p>SPD Lookup Service</p>
+            <p class="subtitle">The Go backend to 1-312-hows-my-driving</p>
+        </a>
+        <a href="https://github.com/SeattleDSA/OpenOversight/">
+            <p>OpenOversight</p>
+            <p class="subtitle">A fork of the Lucy Parsons Labs project for the Salish Sea</p>
+        </a>
+        <a href="https://github.com/SeattleDSA/pra-request-tracker/">
+            <p>Public Records Act Request Tracker</p>
+            <p class="subtitle">An original project for tracking PRA requests</p>
```
Comment (AetherUnbound) [inline src/github.html:37]: _Very original, totally unique ideas, definitely not drawing inspiration from anything that already exists_ 😉

**Review by AetherUnbound — APPROVED** [review]
Looks perfect!

File: src/github.html (hunk 6 above)
Comment (sarayourfriend) [reply src/github.html:37]: Hahaha I wrote that as a nice little in joke for us lol!

**Pull request merged**